pims/bioformats.py

```diff
@@ -54,7 +54,7 @@ def _find_jar():
     return download_jar()
 
 
-def download_jar(version='5.8.2'):
+def download_jar(version='6.4.0'):
```

Comment:
I'd recommend bumping the version number to "6.4" rather than "6.4.0" to take advantages of any minor patch releases between now and 6.5